index.html

```diff
@@ -105,7 +107,7 @@ <h2>Optional</h2>
                     </div>
                 </form>
             </div>
-            <div class="generator" data-type="themes" data-fileName="powercord_manifest.json">
```

Comment:
this is for the download part and should not be changed as manifest files are called like that

Reply:
Changed back to original.